### PR TITLE
feat: implement bert model

### PR DESCRIPTION
**Pull request by sperezacuna**
This PR implemts a basic bert NLP model for sentiment analysis, currently achieving **0.904063 accuracy**. In general:
- Transfer learning from `bert-base-uncased`.
- Train using given datasets (using last 10% of training dataset as validation).

**This is a work in progress**, yet missing:
* [ ] Reliable method of measuring accuracy (F1-macro).
* [ ] Further tweeking to gain accuracy, particularly:
  * [x] Tweek LEARNING_RATE.
  * [ ] Tweek dropout layer.
  * [ ] Change optimizer / scheduler.
  * [ ] Review tokenization procedure.
* [X] Fix requirements.
* [x] Modular & comprehensive code structure.
* [X] Hide warnings.
* [X] Result calculation script (test dataset).

To be discussed:
- Migration to a more complex (and possibly even more efficient model), like [RoBERTa](https://colab.research.google.com/github/DhavalTaunk08/NLP_scripts/blob/master/sentiment_analysis_using_roberta.ipynb#scrollTo=HMqQTafXEaei).

### CODE REVIEW

**Review by crashyspyro — APPROVED** [review]

**Comment by sperezacuna** [comment]
Pending changes opened as issues:
- #2
- #3
- #4
- #5

**Comment by sperezacuna** [comment]
Pending changes opened as issues:
- #2
- #3
- #4
- #5
